### ISSUE TITLE
Track style stuck on full line

### ISSUE THREAD

**Issue by ro-le** [tumic0/GPXSee#457]
After a recent update I am unable to set "track style" to anything but full line. If I try to select (Setting -> Options -> Appearance -> Paths -> Tracks -> Style) any other style (for example dotted line), track remains displayed as full line.

I'm using the latest version 11.5 on Raspbian arm (Qt 5.15.2) and have also tried a fresh Windows install with same results.

**Comment by tumic0** [comment]
TLDR - disable the styles under Data -> Use styles

In 11.5 "data styles" were introduced which changes the way how things are rendered. If you disable the data styles (Data -> Use styles), then you will get exactly the same behaviour as in the pre-11.5 versions, i.e. the line style will be what you set in the preferences. When the data styles are enabled, you should theoretically get the look defined in the data file if the format supports styles and what has been configured in the preferences otherwise. But for the path style this is evidently not the case. I will investigate whether this can be fixed without conflicting with other styling requirements and eventually fix that for 11.6.

**Comment by tumic0** [comment]
Ok, I have fixed (26d2bc4f5d8937353c75aa1ba8e5507d846b5399) the behaviour so, that it will preserve the line style in case no style is provided in the data even when "use styles" is enabled.
